modules/commerce_option_pricing/commerce_option_pricing.install

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * Implements hook_install().
+ */
+function commerce_option_pricing_install() {
+  $entity_type = \Drupal::service('entity_type.manager')->getDefinition('commerce_option_option_value');
+  \Drupal::service('entity.definition_update_manager')->updateEntityType($entity_type);
```

Comment:
Why are we updating the entity at this point? We haven't made any changes to the entity yet right?

Reply:
Ah, it's because of this right: `commerce_option_pricing_entity_base_field_info()`